src/libraries/System.Text.Json/src/System.Text.Json.csproj

```
@@ -9,8 +9,6 @@
     <Nullable>enable</Nullable>
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
     <IsPackable>true</IsPackable>
```

revert the branch specific changes:
```suggestion
    <IsPackable>true</IsPackable>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <ServicingVersion>2</ServicingVersion>
```